src/generate.ts

```diff
@@ -65,7 +65,7 @@ const parseSvg = (svg: string, styles: any) => {
   }
 
   svg = svg.replace(/-([a-z])(?=[a-z\-]*[=\s/>])/g, g => g[1].toUpperCase())
-  svg = svg.replace(/<svg([^>]+)>/, `<svg$1 {...props} height={size} width={size} style={{color}}>`)
+  svg = svg.replace(/<svg([^>]+)>/, `<svg$1 {...props} height={size} width={size} style={{...style,color}}>`)
```

Comment:
Is it necessary to add a default value for style?
```
{...(style || {})}
```

Reply:
Nullish values are ignored for spread operators, so I don't think it's necessary. 

```tsx
{...undefined}
// {}
{...null}
// {}
```

Reply:
Can I have a bump? @unix

Reply:
Version `1.0.3` is out now.

Reply:
That was fast. Many thanks! 🔥